src/mesh/Data.hpp

```diff
@@ -49,8 +49,7 @@ class Data {
       std::string name,
       DataID      id,
       int         dimension,
-      int         spacialDimensions = -1,
-      bool        hasGradient       = false);
+      int         spacialDimensions = -1);
```

Comment:
```suggestion
      int         spatialDimensions = -1);
```
Please fix this on-the-fly if possible. Not sure where this comes from.